test/sql/test_time_fn/T/test_time_fn

```diff
@@ -222,6 +222,14 @@ values ('2010-11-30 23:59:59', '2010-11-30 20:58:59', 'hour'),
        ('2010-11-30 23:59:59', '2010-11-30 20:58:59', 'millisecond');
 select date_diff(d0,d1,t) from test_data_diff order by t;
 
+-- name: test_url_encode
+select url_encode('https://docs.starrocks.io/en-us/latest/quick_start/Deploy');
+select url_encode(NULL);
+
+--name: test_url_decode
+select url_decode('https%3A%2F%2Fdocs.starrocks.io%2Fen-us%2Flatest%2Fquick_start%2FDeploy');
+select url_decode(NULL);
+
```

Comment:
How about URL includes Chinese?

Reply:
It can not in Chinese，because url decode would resolve as other chars

Reply:
ex：